coursier.bzl

```diff
@@ -508,16 +519,17 @@ def _coursier_fetch_impl(repository_ctx):
         primary_url_parts = []
         filepath_parts = artifact["file"].split("/")
         protocol = None
+
         # Only support http/https transports
         for part in filepath_parts:
             if part == "http" or part == "https":
-                    protocol = part
+                protocol = part
```

Comment:
I believe you need to rebase this part on master to add `break` after this line.